drivers/hmis_csv_twenty_twenty_two/spec/models/exporter/filter_coc_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+require_relative './project_setup'
+require_relative './enrollment_setup'
+
+RSpec.describe HmisCsvTwentyTwentyTwo::Exporter::Base, type: :model do
+  include_context '2022 project setup'
+  include_context '2022 enrollment setup'
+
+  describe 'without overrides' do
+    let(:exporter) do
+      HmisCsvTwentyTwentyTwo::Exporter::Base.new(
+        start_date: 1.week.ago.to_date,
+        end_date: Date.current,
+        projects: projects.map(&:id),
+        coc_codes: projects.first.project_cocs.first.CoCCode,
+        period_type: 3,
+        directive: 3,
+        user_id: user.id,
+      )
+    end
+
+    before(:each) do
+      exporter.create_export_directory
+      exporter.set_time_format
+      exporter.setup_export
+      exporter.export_project_cocs
+      exporter.export_enrollment_cocs
+    end
+
+    after(:each) do
+      exporter.remove_export_files
+      exporter.reset_time_format
+      FactoryBot.reload
+    end
+
+    it 'filters ProjectCoC.csv' do
+      csv = CSV.read(File.join(exporter.file_path, 'ProjectCoC.csv'), headers: true)
+      expect(csv.count).to eq 1
+    end
+
+    it 'filters EnrollmentCoC.csv' do
+      csv = CSV.read(File.join(exporter.file_path, 'EnrollmentCoC.csv'), headers: true)
+      expect(csv.count).to eq 0
```

Comment:
So we don't get any EnrollmentCoC ever?  Seems like a smoke test here would also be helpful, especially if the number changed with the override.

Reply:
This is built on the same underlying test framework as the other export tests -- so, we will see enrollment cocs there, and since none of them have a CoC code, this tests to made sure they were filtered out...

But, it was easy enough to add CoCCode to the EnrollmentCoCs...

Reply:
Thanks!